Extensions/harfbuzz/mcclim-harfbuzz.asd

```diff
@@ -0,0 +1,16 @@
+(cl:eval-when (:load-toplevel :execute)
+  (asdf:operate 'asdf:load-op 'cffi-grovel))
```

Comment:
I wonder if it wouldn't make sense to make harfbuzz bindings a separate QL library which we could optionally pull as a dependency

Reply:
also mind my remark regarding dependency on the groveller in base system

Reply:
Yes, the long-term plan is to make both the Fontconfig and Harfbuzz libraries standaline on QL, if they end up being feature-complete. Currently they only implement the features that McCLIM needs, so there is no reason to put them on QL yet.

Reply:
shouldn't it be defsystem-depends-on? also, having such asd file – doesn't it make asdf load automatically groveller regardless we load this system or not?